Function-Specification.md

```diff
@@ -134,8 +136,91 @@ The most popular libraries or study facilities that MUST be displayed:
 
 The user SHOULD be able to choose his/her favorite study facility (i.e. pin it).
 
-#### 2.4.5 Tapping progress bar
+##### 1.4.5 Tapping progress bar
 
 Jump to library seats page.
 
 > TODO: Jump to library seats page.
+> TODO: Pending Design.
+
+
+### 2. Post Page  
+
+![Post Page Screenshot 1](img/post-page-1.png)
+
+#### 2.1 Post Horizontal Slider  
+
+User MAY post information of different categories.  
+This MUST includes:  
+
+- 房屋交友  
+- 二手闲置  
+- 提问回答  
+- 失物招领  
+
+
+##### 2.1.1 Interact with user  
+
+- It is REQUIRED that the horizontal slider only moves when the user scrolls it.
+
+- When user press specific section, that section should be highlighted as shown in the design and the posts displayed should be filtered.
+
+#### 2.2 Post Block
+
+Each *Post* inside post page will be displayed as a post block. A post block should include following elements.
+
+- Time
+- Title
+- Brief content
```

Comment:
I'm assuming this means a limited number of characters extracted from the post's content?

Reply:
> I'm assuming this means a limited number of characters extracted from the post's content?

I guess if the post is too long, only the first part of the post will be displayed on the post feed? and the user can click on the block to see the full content.

Reply:
Makes sense. Say you have a block

> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum vestibulum, dolor at rhoncus molestie, diam leo pellentesque velit, in molestie dui ipsum ac quam. Mauris ac nisi ligula. Nulla dapibus quam eu nisi aliquam commodo. In cursus lorem eu velit elementum pretium. Aenean sagittis metus gravida ipsum feugiat, id venenatis sapien blandit. Fusce quis malesuada nunc. Mauris risus dolor, ultrices vel eleifend in, bibendum volutpat nulla. Vestibulum vel ante et nisi tempus posuere. Vestibulum quis mauris ultrices lectus sagittis mattis id ac mauris. Nunc euismod, erat eget cursus volutpat, sapien magna gravida velit, at mollis nisl tellus sit amet nisl. In ac nisi turpis. Cras ut vestibulum nunc, vel elementum metus. 


Only the first `x` characters would be shown, as in

> Lorem ipsum dolor sit amet, consectetur...
> **Click to view post in detail**